src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -97,8 +97,12 @@ private void OnMemberMouseEnter(object sender, MouseEventArgs e)
             }
 
             FrameworkElement fromSender = sender as FrameworkElement;
-            libraryToolTipPopup.PlacementTarget = fromSender;
-            libraryToolTipPopup.SetDataContext(fromSender.DataContext);
+            var memberVM = fromSender.DataContext as NodeSearchElementViewModel;
```

Comment:
Thanks for taking care of this, we might then want to check for `libraryToolTipPopup != null` too?